app/src/main/java/org/mozilla/fenix/tabtray/TabTrayDialogFragment.kt

```diff
@@ -42,6 +42,7 @@ import org.mozilla.fenix.utils.allowUndo
 class TabTrayDialogFragment : AppCompatDialogFragment() {
     private val tabsFeature = ViewBoundFeatureWrapper<TabsFeature>()
     private var _tabTrayView: TabTrayView? = null
+    private var currOrientation: Int? = null
```

Comment:
Please consider using full naming - `currentOrientation`

Reply:
Fixed and rebased.